src/xpk/commands/batch.py

```diff
@@ -46,11 +48,66 @@ def batch(args: Namespace) -> None:
 
 def submit_job(args: Namespace) -> None:
```

Comment:
Maybe we're missing `--init-image` or don't we need it?

Reply:
Yeah, this is a question. Do we need it?

Reply:
WDYT? @44past4 @mwysokin

Reply:
I see limited value in the option to override this image when running a workload. Therefore for now I would skip this option.

Reply:
Thanks!